Attendee.js

```diff
@@ -0,0 +1,43 @@
+// The Attendee class can be utililized to keep certain information of the attendee, 
+// a class for the user needed to be created in addition to the session class. 
+// The test for this JS class file utilizes Jest.IO
+class Attendee {
+    constructor() {
+        this.sessionId;
+        this.screenName;
+        this.TimeLastPolled;
+    }
+        
+    setSessionId(id) { 
+        this.sessionId = id;
+    }
+
+    setScreenName(name) {
+        this.screenName = name;
+    }
+
+    setTimeLastPolled(time) {
+        this.timeLastPolled = time;
+    }
```

Comment:
Just a thought, instead of having setters, you can pass this information in the constructor